src/documents/tutorial-gettingStartedWithInfusion/BasicComponentCreation-Components.md

```diff
@@ -94,7 +94,7 @@ In the rare case you need to construct a component directly using a JavaScript f
 * plain and **model** components accept a single argument: `options`
 * **view** and **renderer** components accept two arguments: `container` and `options`
 
-_(We'll get into what these arguments are soon.)_
+We'll get into what these arguments are when we talk each type of component later in this tutorial.
```

Comment:
Missing a word here? Such as "about"?

"when we talk **about** each type of component"

Reply:
Done.